cluster/gce/gci/configure-helper.sh

```diff
@@ -1010,6 +1010,21 @@ function start-kube-apiserver {
 
 
   local authorization_mode="RBAC"
+
+  # Create the ABAC file only if it's explicitly requested.
+  if [[ -n "${ENABLE_LEGACY_ABAC}" ]]; then
```

Comment:
This also needs to be `{ENABLE_LEGACY_ABAC_16_ONLY:-}` so that it doesn't bail if that var isn't set.